integration-test/utils/datafactory/schemas/feedback.schema.ts

```diff
@@ -0,0 +1,47 @@
+export const feedbackSectionSchema = {
+    $ref: '#/definitions/feedbackSectionSchema',
+    definitions: {
+      feedbackSectionSchema: {
+        type: 'object',
+        properties: {
+          title: {
+            type: 'string',
+          },
+          feedbacks: {
+            type: 'array',
+            items: [
+              {
+                type: 'object',
+                properties: {
+                  name: {
+                    type: 'string',
+                  },
+                  feedback: {
+                    type: 'string',
+                  },
+                  mentee: {
+                    type: 'boolean',
+                  },
+                  year: {
+                    type: 'object',
+                    properties: {
+                      value: {
+                        type: 'integer',
+                      },
+                      leap: {
+                        type: 'boolean',
+                      },
+                    },
+                    required: ['value', 'leap'],
```

Comment:
`value` and `leap` are not required.
Please, also add `additionalProperties: false,`